readme.md

```diff
@@ -86,6 +86,13 @@ for the complete API.
 
 ## History
 
+Versioning is strictly based on [Semantic Versioning](https://semver.org/)
+
+### 1.0.0 (05-Jul-2019)
+  * BREAKING: all API functions related to hosts require a `hostname` argument
+    now. This way different hostnames listening on the same IP and ports
+    combination do not have an effect on each other.
```

Comment:
the 0.6.x below needs to be removed, and the fix added to 1.0.0.

It also means rendering the docs again.

@locao

Reply:
Done